I think this is ok to do, but you'll need adjust the bitset code instroduced in #30160:
```bash
test/fuzz/bitset.cpp:284:23: error: no matching function for call to 'ReadByte'
    unsigned typdat = ReadByte(buffer) % 8;
                      ^~~~~~~~
test/fuzz/bitset.cpp:16:9: note: candidate function not viable: no known conversion from 'FuzzBufferType' (aka 'span<const unsigned char>') to 'Span<const uint8_t> &' (aka 'Span<const unsigned char> &') for 1st argument
uint8_t ReadByte(Span<const uint8_t>& buffer)
        ^
1 error generated.
```

--- a/src/test/fuzz/bitset.cpp
+++ b/src/test/fuzz/bitset.cpp
@@ -12,8 +12,8 @@
 
 namespace {
 
-/** Pop the first byte from a Span<const uint8_t>, and return it. */
-uint8_t ReadByte(Span<const uint8_t>& buffer)
+/** Pop the first byte from a byte-span, and return it. */
+uint8_t ReadByte(FuzzBufferType& buffer)
 {
     if (buffer.empty()) return 0;
     uint8_t ret = buffer.front();
@@ -23,7 +23,7 @@ uint8_t ReadByte(Span<const uint8_t>& buffer)
 
 /** Perform a simulation fuzz test on BitSet type S. */
 template<typename S>
-void TestType(Span<const uint8_t> buffer)
+void TestType(FuzzBufferType buffer)
 {
     /** This fuzz test's design is based on the assumption that the actual bits stored in the
      *  bitsets and their simulations do not matter for the purpose of detecting edge cases, thus

--- a/src/test/fuzz/fuzz.h
+++ b/src/test/fuzz/fuzz.h
@@ -5,10 +5,9 @@
 #ifndef BITCOIN_TEST_FUZZ_FUZZ_H
 #define BITCOIN_TEST_FUZZ_FUZZ_H
 
-#include <span.h>
-
 #include <cstdint>
 #include <functional>
+#include <span>
 #include <string_view>
 
 /**
@@ -23,7 +22,7 @@
 #define LIMITED_WHILE(condition, limit) \
     for (unsigned _count{limit}; (condition) && _count; --_count)
 
-using FuzzBufferType = Span<const uint8_t>;
+using FuzzBufferType = std::span<const uint8_t>;
 
 using TypeTestOneInput = std::function<void(FuzzBufferType)>;
 struct FuzzTargetOptions {

